test-case/check-ipc-flood.sh

```diff
@@ -34,13 +38,17 @@ func_lib_check_sudo
 dlogi "Check sof debug fs environment"
 [[ "$(sudo file $ipc_flood_dfs|grep 'No such file')" ]] && dlogw "${BASH_SOURCE[0]} need $ipc_flood_dfs to run the test case" && exit 2
 dlogi "Checking ipc flood test!"
-dlogc "sudo bash -c 'echo $lpc_loop_cnt > $ipc_flood_dfs'"
-sudo bash -c "'echo $lpc_loop_cnt > $ipc_flood_dfs'"
 
-sof-kernel-log-check.sh $KERNEL_LAST_LINE
-[[ $? -ne 0 ]] && dloge "Catch error in kernel log" && exit 1
+for i in $(seq 1 $loop_cnt)
+do
+    dlogc "sudo bash -c 'echo $lpc_loop_cnt > $ipc_flood_dfs'"
+    sudo bash -c "'echo $lpc_loop_cnt > $ipc_flood_dfs'"
+
+    sof-kernel-log-check.sh $KERNEL_LAST_LINE
```

Comment:
the `KERNEL_LAST_LINE` will trace from you case start,  
so if you put it into loop you should reset this value  
on the other hand, you can try `sof-kernel-log-check.sh 0` in your loop  
```
    # sof-kernel-log-check script parameter number is 0/Non-Number will force check from dmesg
    sof-kernel-log-check.sh 0
```

I will suggest check from `dmesg` in the loop,  
check full log `/var/log/syslog` at script quit to avoid miss some kernel error

Reply:
just updated as you suggested.
`sof-kernel-log-check.sh 0`